app/src/main/java/org/horaapps/leafpic/activities/SingleMediaActivity.java

```diff
@@ -535,7 +533,7 @@ public void onError() {
                 return true;
 
             case R.id.action_rename:
-                final EditText editTextNewName = new EditText(getApplicationContext());
```

Comment:
What change from `new EditText(getApplicationContext())` and `new EditText(this);`?

Reply:
Using "this" instead of "getApplicationContext()" passes the instance of SingleMediaActivity as the context for EditText. SingleMediaActivity is an instance of ContextThemeWrapper(since Activity extends ContextThemeWrapper). The ContextThemeWrapper is what helps EditText get instantiated with the proper theme. getApplicationContext() is an instance of Context so it does not wrap the editText with the correct theme